modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -432,4 +436,250 @@ public static ReadOnlyBooleanProperty accessibilityActiveProperty() {
+     *         <tr><td>{@code GTK.theme_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_base_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_selected_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_selected_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_base_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_selected_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.theme_unfocused_selected_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.insensitive_bg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.insensitive_fg_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.insensitive_base_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.borders}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.unfocused_borders}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.warning_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.error_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.success_color}</td><td>{@link Color}</td></tr>
+     *         <tr></tr>
+     *     </tbody>
+     * </table>
+     *
+     * @since 22
+     */
+    public interface Preferences extends ObservableMap<String, Object> {
+
+        /**
+         * The platform color scheme, which specifies whether applications should prefer light text on
+         * dark backgrounds, or dark text on light backgrounds. The value of this property defaults to
+         * {@link ColorScheme#LIGHT} if the platform does not report color preferences.
+         *
+         * @return the {@code colorScheme} property
+         * @defaultValue {@link ColorScheme#LIGHT}
+         */
+        ReadOnlyObjectProperty<ColorScheme> colorSchemeProperty();
+
+        ColorScheme getColorScheme();
+
+        /**
+         * The color used for background regions.
+         * <p>
+         * If the platform does not report a background color, this property defaults to {@code Color.WHITE}.
+         *
+         * @return the {@code backgroundColor} property
+         * @defaultValue {@link Color#WHITE}
+         */
+        ReadOnlyObjectProperty<Color> backgroundColorProperty();
+
+        Color getBackgroundColor();
+
+        /**
+         * The color used for foreground elements like text.
+         * <p>
+         * If the platform does not report a foreground color, this property defaults to {@code Color.BLACK}.
+         *
+         * @return the {@code foregroundColor} property
+         * @defaultValue {@code Color.BLACK}
```

Comment:
Is `BLACK` a good default? From what I remember, because some devices have a difficulty with pure black, some dark gray color is used instead.